lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -490,12 +500,13 @@ def record_apply_failure(err, resources: [])
           unidentified_errors << line
         end
       end
+      return unless unidentified_errors.any?
 
-      if unidentified_errors.present? && filenames_with_sensitive_content.any?
+      if filenames_with_sensitive_content.any? { |f| !server_dry_run_validated_resource.include?(f) }
```

Comment:
I think this is the same and easier to read`(filenames_with_sensitive_content - server_dry_run_validated_resource).present?`

Reply:
Ah, that's is an easier way 👍